### DISCUSSION TITLE
Wrong RPATH in macOS LOCAL build

### DISCUSSION THREAD

**Discussion by NaivelyWritten** [eProsima/Fast-DDS#3668] [Q&A]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

`Fast-DDS/install/bin/fastdds discovery` should run fine

### Current behavior

```
dyld[78269]: Library not loaded: @rpath/libfastrtps.2.11.dylib
  Referenced from: <6696D225-E46E-3442-B5F8-55BEA025157D> /Users/<xxx>/Fast-DDS/install/bin/fast-discovery-server-1.0.1
  Reason: tried: '/usr/local/lib/libfastrtps.2.11.dylib' (no such file), '/usr/lib/libfastrtps.2.11.dylib' (no such file, not in dyld cache)
fastdds discovery <discovery-args>

-6
 fast-discovery-server tool not found!
```

### Steps to reproduce

Just compile and try running in latest macOS on M1 Mac according to the docs.

### Fast DDS version/commit

master (`c2b2d4de0c18354e9ea85a5122a74bfbce1e3a2f`)

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

Default configuration, UDPv4 & SHM

### Additional context

macOS Ventura 13.4.1

**Comment by jsan-rt** [comment]
Hi @NaivelyWritten , 

MacOS 13.4 is not one of our [officially supported platforms](https://github.com/eProsima/Fast-DDS/blob/master/PLATFORM_SUPPORT.md) and we currently do not have the required test infrastructure to test this issue.

I'll move it to [Discussions](https://github.com/eProsima/Fast-DDS/discussions) in case the community can provide further insights or tips on the issue.
